src/index.js

```diff
@@ -346,12 +348,12 @@ function getClosenessRanking(testString, stringToRank) {
 function sortRankedItems(a, b) {
   const aFirst = -1
   const bFirst = 1
-  const {rank: aRank, index: aIndex, keyIndex: aKeyIndex} = a
-  const {rank: bRank, index: bIndex, keyIndex: bKeyIndex} = b
+  const {rankedItem: aRankedItem, rank: aRank, keyIndex: aKeyIndex} = a
+  const {rankedItem: bRankedItem, rank: bRank, keyIndex: bKeyIndex} = b
   const same = aRank === bRank
   if (same) {
     if (aKeyIndex === bKeyIndex) {
-      return aIndex < bIndex ? aFirst : bFirst
+      return aRankedItem.localeCompare(bRankedItem)
```

Comment:
Is there ever a case when `aRankedItem` or `bRankedItem` could be anything but a string?

Reply:
Tough to think through, the tests cover every situation i thought of.

- No matches
- Null values
- Array of values
- Array of objects with key
- Nested objects with keys
- Array of objects with multiple keys
- Array of objects with key that points to array

An option as a "just-in-case" would be to add a `typeof` check on the `xRankItem`s and add the indexes back in as a `else` clause but it would not meet 100% test branch coverage.

Reply:
I think we're probably ok. Fixing any issues would be simple and quick so we'll go with this. Thanks!